### DISCUSSION TITLE
Icon for Object Browser etc.

### DISCUSSION THREAD

**Discussion by JonFP** [codefori/vscode-ibmi#2118] [Q&A]
Some time ago Liam showed me how to create a separate sidebar entry for the Object Browser - for me the standard layout is just too cluttered.  When I originally did this (by dragging the Object Browser header to the sidebar) it appeared with a different icon.  Some time in the last few updates that icon disappeared and now I just have two Code for i icons and have to remember which is which.

Can anyone shed any light on how to change the icon so I can more easily differentiate?

**Comment by worksofliam** [comment]
I don’t think there is a way for you to set a custom icon for a view. But, I do believe we need to make sure each view has a unique icon so they can be differentiated in the icon list. 

I will look into this in the upcoming week.

**Comment by JonFP** [comment]
Thanks - for the longest time it had its own icon - but then it magically disappeared!
